### PR TITLE
ESPnet-SPK: Additional models, supplement readme

### PR DESCRIPTION
**Pull request by Jungjee**
## What?

- Add more configured model architectures and well working configurations
  - MFA-Conformer: https://arxiv.org/abs/2203.15249
  - SKA-TDNN: https://ieeexplore.ieee.org/abstract/document/10023305
  - ECAPA-TDNN with WavLM feature
  - x-vector: https://www.danielpovey.com/files/2018_icassp_xvectors.pdf
- Update egs2/TEMPLATE/spk1/README.md for general explanation
- Update egs2/voxceleb/spk1/README.md for models summary and huggingface links

## Why?

- make ESPnet-SPK more competitive

### CODE REVIEW

**Comment by Jungjee** [comment]
@sw005320  I believe this PR is ready to be merged once CI issues are dealt. 
Locally, I checked `test_python` and `test_integration_espnet2.sh` passing.

File: egs2/TEMPLATE/spk1/README.md [hunk 1]
```diff
@@ -67,10 +63,26 @@ See also:
...
+### 6. Speaker embedding extraction
+Extracts speaker embeddings for inference.
+Speaker embeddings belonging to the evaluation set are extracted.
+If `score_norm=true` and/or `qmf_func=true`, cohort set(s) for score normalization and/or quality measure function is also extracted.
+
+### 7. Score calculation
+Calculates speaker similarity scores for an evaluation protocol (i.e., a set of trials).
+One scalar score is calcuated for each trial.
+
+This stage includes score normalization if set with `--score_normtrue`.
```
Comment (sw005320) [inline egs2/TEMPLATE/spk1/README.md:75]:
```suggestion
This stage includes score normalization if set with `--score_norm=true`.
```

File: egs2/voxceleb/spk1/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+| [conf/train_ECAPA_mel.yaml](conf/train_ECAPA_mel.yaml) | 0.856 | 0.0666 | | https://huggingface.co/espnet/voxcelebs12_ecapa_mel |
+| [conf/train_rawnet3.yaml](conf/train_rawnet3.yaml) | 0.739 | 0.0581 | | https://huggingface.co/espnet/voxcelebs12_rawnet3 |
+| [conf/train_SKA_mel.yaml](conf/train_SKA_mel.yaml) | 0.729 | 0.0457 | | https://huggingface.co/espnet/voxcelebs12_ska_mel |
+| [conf/train_ECAPA_wavlm_frozen.yaml](conf/train_ECAPA_wavlm_frozen.yaml) | 0.638 | 0.0499 | ECAPA-TDNN w/ Frozen WavLM | https://huggingface.co/espnet/voxcelebs12_ecapa_frozen|
+| [conf/train_ECAPA_wavlm_joint.yaml](conf/train_ECAPA_wavlm_joint.yaml) | 0.394 | 0.0379 | ECAPA-TDNN w/ Jointly fine-tuned WavLM | https://huggingface.co/espnet/voxcelebs12_ecapa_wavlm_joint |
+| [conf/train_SKA_wavlm_frozen.yaml](conf/train_SKA_wavlm_frozen.yaml) | 0.564 | 0.0548 | SKA-TDNN w/ Frozen WavLM | https://huggingface.co/espnet/voxcelebs12_ska_wavlm_frozen |
+| [conf/train_SKA_wavlm_joint.yaml](conf/train_SKA_wavlm_joint.yaml) |  |  | SKA-TDNN w/ Jointly fine-tuned WavLM | https://huggingface.co/espnet/voxcelebs12_ska_wavlm_joint |
```
Comment (sw005320) [inline egs2/voxceleb/spk1/README.md:14]: Add numbers?

File: egs2/voxceleb/spk1/README.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+- pytorch version: 2.0.1
+
+| | Mean | Std |
+|---|---|---|
+| Target | -0.8015 | 0.1383 |
+| Non-target | 0.0836 | 0.0836 |
+
+| | EER(%) | minDCF |
+|---|---|---|
+|  | 0.739 | 0.05818 |%
```
Comment (sw005320) [inline egs2/voxceleb/spk1/README.md:30]:
```suggestion
|  rawnet3 | 0.739 | 0.05818 |%
```
???

**Review by sw005320 — COMMENTED** [review]
I have minor comments.

**Comment by Jungjee** [comment]
@sw005320  thanks for you detailed checks! I've addressed them.

**Comment by sw005320** [comment]
Cool!
After the CI passes, I'll merge this PR.

**Pull request merged**